src/APIFunctions/2DPrinting.js

```diff
@@ -43,9 +43,16 @@ export function parseRange(pages, maxPages) {
 
 /**
  * Print the page
- * @param {Object} data         Encoded file
- * @returns {PrintApiResponse}  Containing information for if
- *                              the page is printing
+ * @param {Object} data - Encoded file and its configurations
+ * @param {String|undefined} data.raw - Encoded file
```

Comment:
are you sure the uploaded file is represented as a string?

Reply:
Base64 string

Reply:
ok cool

Reply:
@markbrogan13